src/citrine/informatics/processors.py

```diff
@@ -22,6 +25,27 @@ class Processor(Module):
     name = properties.String('config.name')
     description = properties.Optional(properties.String(), 'config.description')
 
+    # TODO: This property should live in AIResourceMetadata. However, since not all modules which
+    # inherit from that class support status_detail, it must live here until they do.
+    status_detail = properties.List(properties.Object(StatusDetail), 'status_detail', default=[],
+                                    serializable=False)
+    """:List[StatusDetail]: a list of structured status info, containing the message and level"""
+
+    @property
+    @deprecated(deprecated_in="1.45.0", removed_in="2.0.0",
+                details="Please use the 'status_detail' property instead.'")
+    def status_info(self):
+        # TODO: This property should live in AIResourceMetadata. However, since not all modules
+        # which inherit from that class support status_detail, it must live here until they do.
+        """[DEPRECATED] :list[str]: human-readable explanations of the status."""
+        return [detail.msg for detail in self.status_detail]
+
+    @status_info.setter
+    def status_info(self, value):
```

Comment:
Does this need a ` @deprecated` annotation as well?  I'm not certain on the scoping of that decorator.